src/styles/shared.module.css

```
@@ -0,0 +1,840 @@
+
+.actionColumnNarrow {
+  width: 120px;
+}
+
+.fullWidthTextarea {
+  width: 100%;
+  min-height: 148px;
+}
+
+.wordBreak {
+  word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)